subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -43,7 +43,8 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
         "1.13.1",
         "1.13.2",
         "1.13.3",
-        "1.13.4"
+        "1.13.4",
+        "1.14-rc1-20180607114235-enterprise_release"
```

Comment:
Why did you add the RC version to the smoke test? I thought we only wanted to dogfood the plugin. I also think the smoke test will fail, since it does not have the right repository configured.

Reply:
as discussed with @etiennestuder we want to have the same coverage as with a production release. We added the repository where the snapshots live to the `libs-snapshots` repository in our Artifactory, so this should work as well.

Reply:
Hmm... I only see the regular plugin portal configured in the test: https://github.com/gradle/gradle/blob/31999d86f21a1f6dfd4ebf5b6f449d0512e445b2/subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy#L90-L112

Let's see if this works.

Reply:
I added the `libs-snapshots` repository in 40cd251 and locally it worked executing it.